uui/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "./.svelte-kit/tsconfig.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "strict": true
+  },
+  // Path aliases are handled by https://kit.svelte.dev/docs/configuration#alias
+  //
+  // from the referenced tsconfig.json - TypeScript does not merge them in
+  "include": [
+    ".svelte-kit/ambient.d.ts",
+    "vite.config.ts",
+    "src/**/*.js",
+    "src/**/*.ts",
+    "src/**/*.svelte",
+    "src/**/*.js",
```

Comment:
Is there a specific need for the duplicate src/**/*.js .ts and .svelte?

Reply:
Nope lol Ill remove those.